### PR TITLE
Extension Gallery Update for 1.22 Release

### PR DESCRIPTION
**Pull request by Charles-Gagnon**

### CODE REVIEW

**Review by Zenix27 — APPROVED** [review]

**Comment by alanrenmsft** [comment]
how about the insider gallery? @Charles-Gagnon

**Comment by alanrenmsft** [comment]
@Zenix27 Thanks for reviewing the PRs. Are you working on Azure Data Studio?

**Comment by Charles-Gagnon** [comment]
@alanrenmsft Already updated.

**Review by alanrenmsft — APPROVED** [review]

**Review by abist — APPROVED** [review]

**Review by kisantia — APPROVED** [review]

**Comment by abist** [comment]
@alanrenmsft @Charles-Gagnon These should be all the changes. If this looks good, I'll push these changes to RC Build.

**Comment by alanrenmsft** [comment]
@abist I just realized you are re-using Charles's branch. lol. you can merge this PR and I believe @joberume will have another PR for kusto extension.

**Comment by abist** [comment]
@alanrenmsft I wanted to consolidate all changes in one PR 😄

**Comment by abist** [comment]
@Charles-Gagnon Could you resolve the conflict and merge? I'm gonna use the same file for the RC build

**Comment by Charles-Gagnon** [comment]
@alanrenmsft I found it easiest to maintain this list in a single PR. As mentioned in the Kusto PR the problem with having everyone update main separately is that if someone needs to actually publish an updated stable gallery (say to fix a breaking issue) they have to make sure to revert the other changes made since those aren't intended to go out until the new version of ADS is released.

File: extensionsGallery.json [hunk 1]
```diff
@@ -3302,6 +3359,63 @@
...
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": "Microsoft.schema-compare"
+								},
+								{
+									"key": "Microsoft.AzDataEngine",
+									"value": ">=1.21.0"
```
Comment (udeeshagautam) [inline extensionsGallery.json:3407]: sorry for late comment. can we please do this also to 1.22 - thanks.

Comment (alanrenmsft) [reply extensionsGallery.json:3407]: I will update this PR

**Comment by abist** [comment]
@Charles-Gagnon Have you been updating the vsix's in the blob storage with the extension updates or should I do that?

**Comment by Charles-Gagnon** [comment]
@abist I've been taking care of that for the arc/azdata extensions so no need to do anything there.

The others you should follow up with the owners on whether they need to be uploaded.

**Review by udeeshagautam — APPROVED** [review]
Please merge to target branch (release/extensions) before uploading. Alan has checked in the ASDE deploy extn also

**Comment by abist** [comment]
@Charles-Gagnon @udeeshagautam @alanrenmsft One of you will need to merge this.

**Comment by udeeshagautam** [comment]
> @Charles-Gagnon @udeeshagautam @alanrenmsft One of you will need to merge this.

I can merge now.

**Pull request merged**